### DISCUSSION TITLE
[Feature] Encrypt user data with Immich password 🔒👪

### DISCUSSION THREAD

**Discussion by Arche151** [immich-app/immich#2514] [Feature Request]
### The feature

I love Immich and I think it’s a great, privacy-friendly option for backing up photos and videos. I want to get my family to switch from iCloud, but I have a concern about the privacy of their data. Even though they can password-protect their images and videos in Immich, I can still access them directly on the disk where they are stored. 

I would like to request a feature that encrypts the user data with their Immich password, so that only they can decrypt and view it, thus offering end-to-end encryption

Because right now, the most privacy-friendly option for my family, would be to stay with E2E-encrypted iCloud for storing their photos and videos.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #450
